qa/suites/fs/upgrade/mds_upgrade_sequence/tasks/3-upgrade-with-workload.yaml

```diff
@@ -11,6 +11,7 @@ upgrade-tasks:
         - ceph config set mon mon_warn_on_insecure_global_id_reclaim false --force
         - ceph config set mon mon_warn_on_insecure_global_id_reclaim_allowed false --force
         - ceph config set global log_to_journald false --force
+        - ceph config set mgr mgr/orchestrator/fail_fs true --force
```

Comment:
shouldn't we add a yaml file to test mds_upgrade_sequence when this option is also "false", which is the default value?

Reply:
I got what you mean here, this yaml also needs to preserve the actual test too right i.e. testing the method of reducing `max_mds` to 1 and upgrading? I'll try to look into this.